css/main.css

```diff
@@ -281,6 +294,90 @@ body {
   height: 5%;
 }
 
+a:hover {
+  text-decoration: none;
+}
+
+.what-can-you-do .nav {
+  display: flex;
+  justify-content: center;
+}
+.what-can-you-do .tab-container {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+}
+.what-can-you-do .tab-container:first-child > span {
+  margin-right: auto;
+}
+.what-can-you-do .tab-container:last-child > span {
+  margin-left: auto;
+}
+.what-can-you-do .nav-tabs {
+  border-bottom: none;
+  margin-bottom: 0.5rem;
+}
+.what-can-you-do .tab-content {
+  width: 100%;
+  height: 20rem;
+  background-color: #e9e3ed;
+  border-radius: 33px;
+  padding: 2rem;
+}
+.what-can-you-do .title {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: #9c8da5;
+}
+.what-can-you-do .tab {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  color: black;
+}
+.what-can-you-do .tab .radio {
+  width: 2.5rem;
+  height: 2.5rem;
+  background-color: #e9e3ed;
+  border-radius: 50%;
+}
+.what-can-you-do .tab.active {
+  animation: fadein 1s;
```

Comment:
nitpick but this is a bit fast/abrupt - think 1.25 looks smoother :)

Reply:
Done !